### PR TITLE
Add more NKIs (non-kitten items) to Robot Finds Kitten

### PR DESCRIPTION
**Pull request by wapcaplet**
#### Summary

SUMMARY: Content "Add more non-kitten items to Robot Finds Kitten"

#### Purpose of change

Since the items are now JSONized (#44942), it is somewhat easier to bring the "Robot Finds Kitten" minigame up to date with the latest [GPL licensed robotfindskitten](https://sourceforge.net/projects/rfk/) item list, and begin expanding with more of our own CDDA-themed items.

#### Describe the solution

Add the "Vanilla NKIs" from [The Ultimate robotfindskitten Fan Site](https://www.crummy.com/software/robotfindskitten/) maintained by original RFK creator Leonard Richardson that were not already present in our list.

Also create a CDDA-themed list, as begun by the moose addition in #27985.

#### Describe alternatives you've considered

Anything but kittens!

#### Testing

Play Robot Finds Kitten, and ensure items from all lists appear (eventually).

#### Additional context

~~WIP with CR - I would happily accept suggestions for more CDDA-themed non-kitten items. Now's your chance to meme!~~

~~Propose your ideas in the comments, or suggest them inline in a code review of the `cdda_themed_nkis` section.~~

Not accepting any more suggestions for this PR. Thanks for playing! `[o#o]  <3 <3 <3   ^o.o^`

image

image

### CODE REVIEW

**Comment by Xaleth** [comment]
"Oooh look! A mininuke (active). Best not touch that..."

File: data/json/snippets/robot_finds_kitten.json [hunk 1]
```diff
@@ -203,8 +204,284 @@
...
+      "Looks like a fully-functional deathmobile.",
+      "A glass flask of Troglobite mutagen.",
+      "Just an ordinary pair of rollerblades.",
+      "Some harmless-looking mushrooms.",
+      "A nest full of containers!  Or is it a container full of nests?",
+      "These socks have storage pouches sewn into them.",
+      "Whatever this is, it would take an extremely long time to wield.",
+      "Camouflaged in the underbrush here is the entrance to an LMOE shelter.",
+      "A box of delicious toast-ems!  Sadly, you are a robot and cannot eat them.",
+      "Enough ordnance to scrub the entirety of New Englang off the face of the earth."
```
Comment (Xaleth) [inline data/json/snippets/robot_finds_kitten.json:448]:
```suggestion
      "Enough ordnance to scrub the entirety of New England off the face of the earth."
```

Comment (wapcaplet) [reply data/json/snippets/robot_finds_kitten.json]: Thanks!

File: data/json/snippets/robot_finds_kitten.json [hunk 2]
```diff
@@ -203,8 +204,289 @@
...
+      "Exactly 37 hot dogs.",
+      "The Panama Papers",
+      "A copy of Don Quixote, slightly eaten.",
+      "A solid gold rocket car.",
+      "Your long lost twin's roommate.",
+      "A pile of very sharp cherry tree petals.",
+      "Some brass laundry tokens.",
+      "A plastic bin containing an assortment of spatulas.",
+      "A power dongle belonging to a long-lost electronic gadget.",
+      "A potato clock that someone forgot to change for Daylight Saving Time."
```
Comment (I-am-Erk) [inline data/json/snippets/robot_finds_kitten.json:489]:
```suggestion
      "A potato clock that someone forgot to change for Daylight Saving Time.",
      "Ninety-nine red balloons.",
      "A bobble-head statue of Genghis Khan winking at you.",
      "Your sense of dignity.",
      "The One Paperclip, forged in the fires of Mount Doom.",
      "That one screw you were missing to finish building an Ikea desk three years ago.",
      "A kitten!  No... wait, it's a turkey."
```

File: data/json/snippets/robot_finds_kitten.json [hunk 3]
```diff
@@ -203,8 +204,295 @@
...
+      "Some brass laundry tokens.",
+      "A plastic bin containing an assortment of spatulas.",
+      "A power dongle belonging to a long-lost electronic gadget.",
+      "A potato clock that someone forgot to change for Daylight Saving Time.",
+      "Ninety-nine red balloons.",
+      "A bobble-head statue of Genghis Khan winking at you.",
+      "Your sense of dignity.",
+      "The One Paperclip, forged in the fires of Mount Doom.",
+      "That one screw you were missing to finish building an Ikea desk three years ago.",
+      "A kitten!  No... wait, it's a turkey."
```
Comment (ZhilkinSerg) [inline data/json/snippets/robot_finds_kitten.json:495]:
```suggestion
      "A kitten!  No… wait, it's a turkey."
```

**Pull request merged**